tests/unit/amo/components/TestCollection.js

```diff
@@ -723,9 +712,7 @@ describe(__filename, () => {
     const wrapper = renderComponent({ store, _config: fakeConfig });
 
     const editLink = wrapper.find('.Collection-edit-link').find(Button);
-    expect(editLink).toHaveLength(1);
-    expect(editLink).toHaveProp('to',
-      `/collections/${defaultUser}/${defaultCollectionDetail.slug}/edit/`);
+    expect(editLink).toHaveLength(0);
```

Comment:
As above, this test just needs to be changed to use a user who owns the collection.